.Rbuildignore

```diff
@@ -1,6 +1,7 @@
 ^jmpost\.Rproj$
 ^\.Rproj\.user$
 ^LICENSE\.md$
+^LICENSE$
```

Comment:
I would suggest to completely remove the `LICENSE` file, note that in my experience CRAN reviewers for new packages will also complain about these.

Reply:
fully agree. and we have lots of Stats Eng packages without `LICENSE` file.